semantic-conventions/src/tests/data/markdown/deprecated/expected.md

```diff
@@ -4,12 +4,12 @@
 <!-- semconv http -->
 | Attribute  | Type | Description  | Example  | Required |
 |---|---|---|---|---|
-| `http.method` | string | HTTP request method. | `GET`<br>`POST`<br>`HEAD` | Yes |
+| `http.method` | string | HTTP request method. | `GET` or `POST` or `HEAD` | Yes |
```

Comment:
```suggestion
| `http.method` | string | HTTP request method. | `GET`, `POST`, `HEAD` | Yes |
```

I would prefer commas. Using multiple `or`s sounds wrong.

Reply:
On second thought, I agree. Reading or multiple times sounds weird.
Maybe `or` only for two options and `,` for multiple?

Reply:
If there is a concern that commas might make people think of arrays, I suggest: Use `or` only between the last two items of the list (like you normally do in English); or use semicolons instead of commas as separators.

Reply:
I think putting a smart join in here (that uses `or` between two things, commas between more than two, and an additional `or` between the final two) seems like too much for this tool.  And I'd really prefer not to start a conversation about whether to use an oxford comma. 

I vote for semicolons. 😄

Reply:
@open-telemetry/specs-approvers what would you suggest to use?

Reply:
The table header should say _Example**s**_ instead of _Example_ to make clear those are multiple, separate examples (still fine if there is only one example given).
I'd say commas and one _or_ (rather without oxford comma), just commas or just semicolons are all fine. When it says example**s** on top and also mentions the type, even commas should not confuse readers. Multiple *or*s really look odd, though.

Reply:
Let's use semicolons!

Reply:
Done in 8053f628e25e7486766adb8a837d19ecb160a5cb